serverless/pytorch/shiyinzhang/iog/nuclio/function.yaml

```diff
@@ -7,6 +7,7 @@ metadata:
     spec:
     framework: pytorch
     min_pos_points: 1
+    min_neg_points: 0
```

Comment:
It looks the method can have negative points:

```
Clicks representations: We use the same clicks representation as DEXTR[46] by centering a 2D Gaussian around
each click, creating two separate heatmaps for foreground
and background clicks. The resulting heatmaps are concatenated with the RGB input image to form a 5-channel
input for the network. Similar to [46], the bounding box is
first relaxed by several pixels to include context, followed
by cropping to focus on the object-of-interest (Figure 2(d)).
```

Reply:
```
Although our IOG approach requires only three clicks
to perform segmentation, our framework naturally supports interactive adding of new foreground and background
clicks for further refinement if the user is not satisfied with
the current segmentation output. To achieve this, we append a lightweight branch before the PSP module to accept the two-channel Gaussian heatmaps encoding all the
foreground and background clicks
```

Reply:
Okay, it has in the implementation.
``min_neg_poinst: 0`` means that method supports negative points, but they are not obligatory.